frontends/concrete-python/examples/levenshtein_distance/IBAN_name_check.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bf2a3e8d-bf57-4841-a863-1eab96c91373",
+   "metadata": {},
+   "source": [
+    "# Comparing encrypted IBAN names\n",
+    "\n",
+    "When doing a transfer between Bank A and Bank B, Bank B has the obligation to check that the IBAN and the name of the recipient match. This is essential to combat frauds (fraudster impersonating someone else) and to avoid misdirected payments. Bank B would usually not reject a transfer if the name is close enough but doesn’t match exactly the recipient’s actual name. This is essential to make room for small spelling mistakes considering the impact of a rejected transfer (days / weeks of delays that can harm a business or a buyer, extra costs to handle the error, …). It is therefore important for Bank A to pre-check the name and inform the sender that the name is likely not matching, before initiating the transfer. For privacy reason however, it's better to do this pre-check over encrypted names.\n",
+    "\n",
+    "In this small tutorial, we show how to use our TFHE Levenshtein distance computations to perform such a privacy-preserving check, very simply and directly in Python. This tutorial can be easily configured, to change for example the way strings are normalized before encryption and comparison. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc96a80f-0b14-4e64-a33f-31a60351453d",
+   "metadata": {},
+   "source": [
+    "## Importing our FHE Levenshtein computations\n",
+    "\n",
+    "One can have a look to this file to see how the FHE computations are handled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "56ba9e20-ca46-4aa6-a0f7-86ca13480a52",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from levenshtein_distance import *\n",
+    "from time import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "494cd58c-ea28-4547-92c5-80ed4ba83964",
+   "metadata": {},
+   "source": [
+    "## Define the comparison functions\n",
+    "\n",
+    "FHE computation will happen in `calculate_and_return`, if `fhe_or_simulate` is set to `fhe`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "2410b6b3-0c21-4178-b8dc-f734ec0afd40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Arguments:\n",
+    "    \"\"\"Fake an argument parser, to be able to reuse the functions from levenshtein_distance.py.\"\"\"\n",
```

Comment:
I think we should refactor levenshtein distance example to be free of "arguments" for construction. Instead we can pass `max_string_length`, `show_mlir`, and `show_optimizer` directly to the class.

Reply:
We can but is it really needed?

Reply:
IMO, yes. We'll add tests/benchmarks as well, and it's worse to create this fake arguments class. Simple kwargs would be better overall.

Reply:
ok let me do it

Reply:
done